extensions/answer_summarizers/models_test.py

```diff
@@ -45,10 +45,10 @@ def _create_answer_dict(
         Args:
             answer: dict(str, *). The answer in dict format.
             time_spent_in_card: float. The time spent (in sec) in each card. By
-                default, it's 3.2 sec.
+                default, it's '3.2 sec'.
```

Comment:
Oh I meant to do this only for the below one (as it was a string). Here I think it is fine without quotes

Reply:
Done